.rubocop.yml

```diff
@@ -139,11 +136,75 @@ Style/DateTime:
     - "rakelib/evss.rake"
     - "lib/evss/auth_headers.rb" #DateTime is needed for date formatting here
 
-
 # Skipping for now, should revisit:
 Rails/HasManyOrHasOneDependent:
   Exclude:
     - "app/models/education_benefits_claim.rb"
     - "app/models/saved_claim/education_benefits.rb"
     - "app/models/saved_claim.rb"
     - "app/models/terms_and_conditions.rb"
+
+# Using .empty? with the safe navigation operator seems to be the correct call in this situation
+Lint/SafeNavigationWithEmpty:
+  Exclude:
+    - 'lib/hca/configuration.rb'
+
+# The suggestions here don't seem to make sense in most cases. Skip for now.
+Naming/MemoizedInstanceVariableName:
+  Enabled: false
+
+# The suggestions here appear that they would break intended behavior. Skip for now.
+Naming/RescuedExceptionsVariableName:
+  Exclude:
+    - 'app/controllers/concerns/accountable.rb'
+    - 'app/workers/central_mail/submit_form4142_job.rb'
+    - 'app/workers/evss/disability_compensation_form/job_status.rb'
+    - 'app/workers/evss/disability_compensation_form/submit_form0781.rb'
+    - 'app/workers/evss/disability_compensation_form/submit_form8940.rb'
+    - 'lib/benchmark/performance.rb'
+    - 'lib/common/client/concerns/monitoring.rb'
+    - 'lib/common/client/middleware/response/facility_validator.rb'
+    - 'lib/common/client/middleware/response/json_parser.rb'
+    - 'lib/ihub/appointments/service.rb'
+    - 'lib/saml/user_attributes/id_me.rb'
+    - 'lib/search/service.rb'
+    - 'lib/vet360/contact_information/service.rb'
+
+# It was determined these suggestions had the potential to change behavior. 
+Rails/ActiveRecordAliases:
+  Enabled: false
+
+# Use of private inline seems to be for purpose of grouping related methods.
+# Use of module_function inline appears to be correct usage according to Ruby styleguide.
+# Skip for now.
+Style/AccessModifierDeclarations:
+  Exclude:
+    - 'lib/sm/client.rb'
+    - 'modules/va_facilities/lib/va_facilities/api_serialization.rb'
+    - 'modules/va_facilities/lib/va_facilities/api_serialization_v1.rb'
+
+# The use of variables such as n to represent number, x and y for x-axis and y-axis, el for element, 
+# v for value, i for index, and e for error are all standard abbreviations and should be excluded.
+Naming/UncommunicativeMethodParamName:
+  Exclude:
+    - 'config/initializers/integration_recorder.rb'
+    - 'lib/central_mail/datestamp_pdf.rb'
+    - 'lib/emis/responses/response.rb'
+    - 'lib/mvi/responses/profile_parser.rb'
+    - 'lib/mvi/service.rb'
+    - 'lib/pdf_fill/hash_converter.rb'
+    - 'lib/sentry_logging.rb'
+    - 'modules/vba_documents/app/workers/vba_documents/upload_processor.rb'
+    - 'rakelib/mvi.rake'
+
+# These do not seem to actually be using Kernel#open so should be excluded.
+Security/Open:
```

Comment:
They are using `Kernel#open`.  When you just call `open` (instead of something like `File.open` it is being called on `Kernel`.
Glancing at the files listed below, I believe you can fix them by changing `open` calls to `File.open` in each file.

Reply:
It looks like I was putting too much trust in SublimeText. It was listing the 'open' being used as a method coming from line 72 of evss_claim_base_serializer.rb